index.html

```diff
@@ -1913,6 +1919,67 @@
 
 </aside>
 
+## Transparency
+
+<div class="practice">
+  <span class="practicelab" id="transparency-when-requested">
+    When accessing data or requesting permission, [=sites=] (and other [=actors=]) should provide
+    [=people=] with relevant information about its use and [=user agents=] should help present and
```

Comment:
I'm not sure exactly how to best fix this, but I stumbled over the pairing of "its use" with both "accessing data" and "requesting permission". We could make it "requesting permission to access data". Or "information about the site's plans" if the permission might also be about things like storing data. Or I'm not sure that "requesting permission" belongs here, since transparency is about what the sites do with the thing they got permission for.

Reply:
Tried to clarify this in recent commit, "with relevant explanatory information about the use of data".